assignments/this.js

```diff
@@ -1,10 +1,10 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Window binding is the default binding of "this". "This" will always refer to window unless it is implicitly or explicitly bound to something else. In node, "this" seems to be bound to an empty object.
```

Comment:
Because in node there's no `window` object; it could be considered a gotcha, but you won't really have to worry about interacting with it (so long as you're mindful to not do so accidentally).